inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java

```diff
@@ -129,11 +129,12 @@ public InlongGroupContext init() throws Exception {
             }
         }
         String formDataNew = formDataJson.toString();
-        NewGroupProcessForm newGroupProcessForm = JsonUtils.parseObject(
-                formDataNew, NewGroupProcessForm.class);
-        Preconditions.checkNotNull(newGroupProcessForm, "NewGroupProcessForm cannot be null");
-        groupContext.setInitMsg(newGroupProcessForm);
-        WorkflowResult startWorkflowResult = workFlowClient.startInlongGroup(taskId, newGroupProcessForm);
+        ApplyGroupProcessForm groupProcessForm = JsonUtils.parseObject(
+                formDataNew, ApplyGroupProcessForm.class);
+        Preconditions.checkNotNull(groupProcessForm, "ApplyGroupProcessForm cannot be null");
+        groupContext.setInitMsg(groupProcessForm);
+        assert groupProcessForm != null;
+        WorkflowResult startWorkflowResult = workFlowClient.startInlongGroup(taskId, groupProcessForm);
```

Comment:
I think it should be "workflowClient"

Reply:
Yeah, I think so.
But this class belongs to the client module and should change in another single PR.

Reply:
private final WorkflowClient workFlowClient;
I was referring to this member declaration in the class you have modified.